src/base/bittorrent/session.h

```diff
@@ -59,11 +59,21 @@ class BandwidthScheduler;
 class Statistics;
 class ResumeDataSavingManager;
 
+// These actions were tied to the combobox's indexes
```

Comment:
Please don't add any dependencies from UI (or other client components), even in comments.

Reply:
Suffice it to mention that the values of the enum items should remain unchanged when adding (inserting) new ones, so as not to break the existing user settings.

Reply:
Changed.